gpii/node_modules/matchMakerFramework/src/MatchMakerFramework.js

```diff
@@ -81,6 +80,158 @@
             event.fire(matchData, matchMakerPayload);
         });
     };
+	
+    /*
+	 * Returns a string indicating the matchmaker to select from configuration. Depends on the matchmaker payload.
+     */
+    gpii.matchMakerFramework.selectMatchMaker = function (matchMakerPayload) {
+		var result = gpii.matchMakerFramework.GetPreferredMatchMaker(matchMakerPayload);
+		return (result) ? result : gpii.matchMakerFramework.GetCapableMatchMaker(matchMakerPayload);
+    };
+
+    /*
+	 * Returns if a matchmaker payload contains preferences for selecting a matchmaker.
+	 * Returns NULL if no matchmaker preference was found.
+     */
+	gpii.matchMakerFramework.GetPreferredMatchMaker = function (matchMakerPayload) {
+		// Constants
+		var USER_DEFINED_MATCHMAKER_PREF0 = "http://registry.gpii.org/common/matchMakerType";
+		var USER_DEFINED_MATCHMAKER_PREF1 = "http://registry.gpii.net/common/matchMakerType";
```

Comment:
Is it me, or are these two identical? It looks like one of them safely can be removed